src/main/java/org/jitsi/videobridge/octo/OctoEndpoint.java

```diff
@@ -78,6 +78,25 @@ public void requestKeyframe(long mediaSsrc)
         }
     }
 
+    @Override
+    public void requestKeyframe()
+    {
+        // just making sure the tentacle hasn't expired
+        OctoTentacle tentacle = getConference().getTentacle();
+        if (tentacle != null)
+        {
+            long firstVideoSsrc = receiveSsrcs.entrySet().stream()
+                    .filter(entry -> entry.getValue().equals(MediaType.VIDEO))
+                    .findFirst()
+                    .map(Map.Entry::getKey)
+                    .orElse(-1L);
+            if (firstVideoSsrc >= 0)
+            {
+                requestKeyframe(firstVideoSsrc);
```

Comment:
i could just call `tentacle.requestKeyframe(firstVideoSsrc);` here, but i thought it would be better to call the existing code.  it's unfortunate that then we have to look up the tentacle twice, though, so definitely open to changing this if you think it'd be better.